### ISSUE TITLE
Errors when creating/reading plugin manifest file

### ISSUE THREAD

**Issue by DRAGKILLS** [KygekDev/PluginMaker#1]
![image](https://user-images.githubusercontent.com/70339357/146724470-d9d5f190-2fef-4592-9b3e-260bc3739578.png)

**Comment by KygekDev** [comment]
Yes, this tool is buggy and I'm aware of this issue. It needs updating to make it functional. Unfortunately, I don't think I will have time to fix the bugs.

If you want to get it fixed, you have two options:
1. Contact the original developer: @GamakCZ (e.g. make an issue in the [original repository](https://github.com/GamakCZ/PluginMaker).
2. Ask someone to **fork** this repository (**not the original!**) and **submit a PR** to this repository.

By the way, thank you very much for reporting the bug. If you have any questions, feel free to ask in this issue.
